pkg/codegen/go/gen_test.go

```diff
@@ -145,166 +175,166 @@ func (mocks) Call(args pulumi.MockCallArgs) (resource.PropertyMap, error) {
-			rubberTree, err := tree.NewRubberTree(ctx, "blah", &tree.RubberTreeArgs{
-				Container: plant.ContainerArgs{
-					Color:    pulumi.String("Magenta"),
-					Material: pulumi.String("ceramic"),
-					Size:     plant.ContainerSize(22),
-				},
-				Farm: tree.Farm_Plants_R_Us,
-				Type: tree.RubberTreeVarietyBurgundy,
-			})
-			require.NoError(t, err)
-			require.NotNil(t, rubberTree)
-			var wg sync.WaitGroup
-			wg.Add(1)
-			pulumi.All(
-				rubberTree.URN(),
-				rubberTree.Container.Material(),
-				rubberTree.Container.Color(),
-				rubberTree.Container.Size(),
-				rubberTree.Type,
-			).ApplyT(func(all []interface{}) error {
-				urn := all[0].(pulumi.URN)
-				material := all[1].(*string)
-				color := all[2].(*string)
-				size := all[3].(*plant.ContainerSize)
-				typ := all[4].(tree.RubberTreeVariety)
-				assert.Equal(t, *material, "ceramic", "unexpected material on resource: %v", urn)
-				assert.Equal(t, *color, "Magenta", "unexpected color on resource: %v", urn)
-				assert.Equal(t, *size, plant.ContainerSize(22), "unexpected size on resource: %v", urn)
-				assert.Equal(t, typ, tree.RubberTreeVarietyBurgundy, "unexpected type on resource: %v", urn)
-				wg.Done()
-				return nil
-			})
-			wg.Wait()
-			return nil
-		}, pulumi.WithMocks("project", "stack", mocks(1))))
-	})
-
-	t.Run("EnumOutputs", func(t *testing.T) {
-		require.NoError(t, pulumi.RunErr(func(ctx *pulumi.Context) error {
-			rubberTree, err := tree.NewRubberTree(ctx, "blah", &tree.RubberTreeArgs{
-				Container: plant.ContainerArgs{
-					Color:    plant.ContainerColor("Magenta").ToContainerColorOutput().ToStringOutput(),
-					Material: pulumi.String("ceramic").ToStringOutput(),
-					Size:     plant.ContainerSize(22).ToContainerSizeOutput(),
-				},
-				Farm: tree.Farm_Plants_R_Us.ToFarmPtrOutput().ToStringPtrOutput(),
-				Type: tree.RubberTreeVarietyBurgundy.ToRubberTreeVarietyOutput(),
-			})
-			require.NoError(t, err)
-			require.NotNil(t, rubberTree)
-			var wg sync.WaitGroup
-			wg.Add(1)
-			pulumi.All(
-				rubberTree.URN(),
-				rubberTree.Container.Material(),
-				rubberTree.Container.Color(),
-				rubberTree.Container.Size(),
-				rubberTree.Type,
-			).ApplyT(func(all []interface{}) error {
-				urn := all[0].(pulumi.URN)
-				material := all[1].(*string)
-				color := all[2].(*string)
-				size := all[3].(*plant.ContainerSize)
-				typ := all[4].(tree.RubberTreeVariety)
-				assert.Equal(t, *material, "ceramic", "unexpected material on resource: %v", urn)
-				assert.Equal(t, *color, "Magenta", "unexpected color on resource: %v", urn)
-				assert.Equal(t, *size, plant.ContainerSize(22), "unexpected size on resource: %v", urn)
-				assert.Equal(t, typ, tree.RubberTreeVarietyBurgundy, "unexpected type on resource: %v", urn)
-				wg.Done()
-				return nil
-			})
-			wg.Wait()
-			return nil
-		}, pulumi.WithMocks("project", "stack", mocks(1))))
-	})
-}
+// func TestEnumUsage(t *testing.T) {
```

Comment:
TODO I need to move these into in-place Go tests so that normal "test" pass can pick them up.